addons/nightvision/functions/fnc_blending.sqf

```diff
@@ -23,25 +23,19 @@
 
 if (!hasInterface) exitWith {};
 
-private ["_vehicle", "_weapon", "_ammo", "_magazine", "_player"];
+params ["_vehicle", "_weapon", "", "", "_ammo", "_magazine"];
 
-_vehicle = _this select 0;
-_weapon = _this select 1;
-_ammo = _this select 4;
-_magazine = _this select 5;
-
-_player = ACE_player;
```

Comment:
I personally prefer when a function uses `_unit` throughout the file and defines it either on the top of the file with `_unit = ACE_player` or preferably inherits it from wherever it was called from.